README.md

```diff
@@ -21,35 +21,35 @@ Color scheme : https://colorhunt.co/palette/269111
 `py -m venv env`(on Windows) or `python3 -m venv env`(on Mac and Linux)
 * Activate the Virtual Environment : `.\env\Scripts\activate`(for Windows) or `source env/bin/activate`(on Mac and Linux)
 * Then `fork` this repository to your account
-* Clone it in your local machine using command `git clone url`
+* Clone it in your local machine using the command `git clone url`
 * Navigate to the project directory and install the project requirements using `pip install -r requirements.txt`
-* After a copy of the project is made in your machine , follow the `Explore` section to run the project in your local server.
-* All pages are present in the `templates` folder , project files in the `pro_act` folder.
-* Make the required changes and run it in the server to see the working.
+* After a copy of the project is made in your machine, follow the `Explore` section to run the project on your local server.
+* All pages are present in the `templates` folder, project files in the `pro_act` folder.
+* Make the required changes and run them on the server to see them working.
 * `git add .` to add all the changes and `git add filename.txt` to add changes to a particular file.
 * `git commit -m "message"` it is recommended to commit with a message to tell what are the things you have changed.
-* `git push` to push all changes to main branch from local branch.
+* `git push` to push all changes to the main branch from the local branch.
 ### PS: Virtual Environment can be installed on windows by `py -m pip install --user virtualenv`
 
 
 
 
 <h2 align="center"> Explore 🌏</h2>
 
-Try it out by installing Django. (It is required to have Python installed , if not then download it from : [here](https://www.python.org/downloads/))
+Try it out by installing Django. (It is required to have Python installed, if not then download it from [here](https://www.python.org/downloads/))
 
     python -m pip install Django
     
     
-After installation of Django, open this file in any text editor and in the teminal type the following commandes to migrate the file . 
+After installation of Django, open this file in any text editor, and in the terminal type the following command to migrate the file. 
 Migrate the files:
 
     python manage.py makemigrations
     python manage.py migrate
 
 And then start the server (default: http://localhost:8000)
 
-    python manage.py runserver
+    python manage.py run server
```

Comment:
Please change this to `runserver` because it is a command & It won't work if it contains whitespace.